docs/docs/getting_started/first_profile.md

```diff
@@ -5,51 +5,51 @@ sidebar_position: 60
 
 # Creating your first profile
 
-Minder uses [profiles](../how-to/create_profile.md) to specify common,
-consistent configuration which should be enforced on all registered
-repositories.  In this tutorial, you will register a GitHub repository and
-create a profile that indicates whether secret scanning is enabled on the
-registered repositories.
+Once you have registered repositories, you can create a profile that specifies the common, consistent configuration that you expect your your repositories to comply with. 
 
 ## Prerequisites
 
-* [The `minder` CLI application](./install_cli.md)
-* [A Minder account](./login.md)
-* [An enrolled GitHub token](./login.md#enrolling-the-github-provider) that is either an Owner in the organization or an Admin on the repositories
-* [One or more repositories registered with Minder](./register_repos.md)
+Before you can create a profile, you should [register repositories](register_repos).
```

Comment:
**nit:** this is not strictly a prerequisite, but I guess this is the desired experience for a user going through the docs.

Reply:
True. 😁  I worded it as _should_ because of that. I didn't really want to untangle all the details in the getting started page. 😁